dapps/marketplace/src/components/Reviews.js

```diff
@@ -173,26 +182,41 @@ export default class Reviews extends Component {
 require('react-styl')(`
   .reviews
     margin-bottom: 2rem
+    h3
+      font-family: Poppins
+      font-size: 1.5rem
+      font-weight: 500
+      font-style: normal
```

Comment:
It seems like having all these 'normal' declarations shouldn't be needed... isn't the default normal?

Reply:
Actually, it inherits from parent/ancestor elements. If none of the elements have that property set, it should be `normal`. In some places, it is not necessary. I'll do a clean up of that.